.github/workflows/CI.yml

```diff
@@ -22,24 +22,13 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # Oldest supported versions
-          # NOTE: renable CUDA 10.2 when it supported by NNPOps (https://github.com/conda-forge/nnpops-feedstock/pull/8)
-          - name: Linux (CUDA 11.0, Python 3.7, PyTorch 1.11)
-            os: ubuntu-22.04
-            cuda-version: "11.0.3"
-            gcc-version: "8.5.*"
-            nvcc-version: "11.0"
-            python-version: "3.7"
-            pytorch-version: "1.11.*"
-
-          # Latest supported versions
-          - name: Linux (CUDA 11.8, Python 3.10, PyTorch 2.0)
+          - name: Linux (CUDA 11.8, Python 3.10, PyTorch 2.1)
             os: ubuntu-22.04
             cuda-version: "11.8.0"
             gcc-version: "10.3.*"
             nvcc-version: "11.8"
             python-version: "3.10"
-            pytorch-version: "2.0.*"
+            pytorch-version: "2.1.*"
```

Comment:
There is pytorch 2.3 in conda-forge.

Reply:
I just tried 2.3, but it fails to install.

```
Could not solve for environment specs
The following packages are incompatible
├─ __cuda is requested and can be installed;
├─ python 3.10**  is installable with the potential options
│  ├─ python [3.10.0|3.10.10|...|3.10.9], which can be installed;
│  └─ python [3.10.0|3.10.1|...|3.10.9] would require
│     └─ python_abi 3.10.* *_cp310, which can be installed;
└─ pytorch-gpu 2.3**  is not installable because there are no viable options
   ├─ pytorch-gpu 2.3.0 would require
   │  └─ pytorch 2.3.0 cuda118_py39hd44be3b_300, which requires
   │     ├─ python >=3.9,<3.10.0a0 , which conflicts with any installable versions previously reported;
   │     └─ python_abi 3.9.* *_cp39, which conflicts with any installable versions previously reported;
   └─ pytorch-gpu 2.3.0 would require
      └─ pytorch 2.3.0 cuda120_py38heb61fd4_300, which requires
         └─ cuda-version >=12.0,<13 , which requires
            └─ __cuda >=12 , which conflicts with any installable versions previously reported.
```

Reply:
There is no 2.2, they skipped directly to 2.3.0 after 2.1.2

Reply:
Pytorch 2.3 requires CUDA >=12 it seems.

Reply:
Going forward, have proposed some suggestions that may help alleviate these issues ( https://github.com/openmm/openmm-torch/issues/146 ). Also this may help with moving to CUDA 12 when that happens